Crust Crawler Arms/read_joint_positions/read_joint_positions.ino

```diff
@@ -34,8 +34,10 @@ void loop(){
   // and send it to the PC.
   for(int i = 1; i < 6; ++i){
     int pos = GetPosition(i);
-    String report;
-    report = i + ": " + pos;
+    String servo=String(i);
+    String text=String(": ");
+    String posit=String(pos);
+    String report = String(servo+text+posit);
```

Comment:
I don't like the style here. There is really no reason to make three String objects and concatenate them in a String constructor.
If concatenation of integers in strings does not work, why not just change line 38 to:
`report = String(i) + String(": ") + String(pos)`